net/sfeed/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           legacysupport 1.1
+PortGroup           makefile 1.0
+
+name                sfeed
+version             1.1
+revision            0
+license             ISC
+
+categories          net
+maintainers         {sikmir @sikmir} openmaintainer
+description         RSS and Atom parser (and some format programs)
+long_description    ${description}
+
+homepage            https://git.codemadness.org/${name}/
+platforms           darwin
```

Comment:
`platforms darwin` is now implicit and no longer necessary to actually specify in the Portfile.